crm/modules/member/form.inc.php

```diff
@@ -507,7 +507,8 @@ function member_membership_delete_form ($sid) {
         , 'method' => 'post'
         , 'command' => 'member_membership_delete'
         , 'hidden' => array(
-            'sid' => $membership['sid']
+            'sid' => $membership['sid'],
+            'cid' => $membership['cid']
```

Comment:
not sure what the benefit of adding this here is, as `$membership['cid']` isn't used at all by `command_member_membership_delete` or `member_membership_delete`